frontend/src/stores/schedule-store.ts

```diff
@@ -1,10 +1,14 @@
+import { i18n } from '@/composables/i18n';
 import { defineStore } from 'pinia';
-import { ref, computed } from 'vue';
+import { ref, computed, inject } from 'vue';
 import { useUserStore } from '@/stores/user-store';
+import { dateFormatStrings } from '@/definitions.js';
 import { Fetch, Schedule, ScheduleListResponse } from '@/models';
 
 // eslint-disable-next-line import/prefer-default-export
 export const useScheduleStore = defineStore('schedules', () => {
+  const dj = inject('dayjs');
```

Comment:
This needs to be typed, otherwise TS won't know that dj is a Dayjs callable

```suggestion
  const dj = inject<(date?: ConfigType) => Dayjs>('dayjs');
```

with

```js
import { Dayjs, ConfigType } from 'dayjs';
```